### ISSUE TITLE
Publish new release

### ISSUE THREAD

**Issue by Ryuno-Ki** [Ryuno-Ki/webmention-tools#16]
Hi, @vrypan,

I'd like to use this tool on my homepage + blog, so I'd kindly ask you to publish a new releae. C.f.
https://github.com/vrypan/webmention-tools/issues/13#issuecomment-370930871

I tested it using `pip install git+https://github.com/vrypan/webmention-tools.git@master`. `urlinfo` option of CLI is working, but `send` fails (PR will come. Certification warning for https://webmention.rocks/test/1 + BeautifulSoup parser warning).

**Comment by Ryuno-Ki** [comment]
After working on #18 I was able to get it somewhat working.

I don't understand though, why this fails:
```sh
$ webmention-tools send https://webmention.rocks/test/1 https://jaenis.ch/blog/experimenting-with-webmentions/ --debug
Sending webmention from https://webmention.rocks/test/1 to https://jaenis.ch/blog/experimenting-with-webmentions/... 
$VENV/lib/python3.6/site-packages/urllib3/connectionpool.py:858: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
$VENV/lib/python3.6/site-packages/urllib3/connectionpool.py:858: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
Failed
{'code': 'RECEIVER_ERROR',
 'http_status': 201,
 'location': 'https://webmention.io/jaenis.ch/webmention/qOGZzOCWm_7pJKdkNj3e',
 'request': 'POST https://webmention.io/jaenis.ch/webmention (with '
            'source=https://webmention.rocks/test/1, '
            'target=https://jaenis.ch/blog/experimenting-with-webmentions/)',
 'source': 'https://webmention.rocks/test/1',
 'status': 'queued',
 'summary': 'Webmention was queued for processing',
 'target': 'https://jaenis.ch/blog/experimenting-with-webmentions/'}
```

201 means created. Why shall that constitute an error?

**Comment by Ryuno-Ki** [comment]
Hm,
```sh
webmention-tools send https://adactio.com/journal/6469 https://jaenis.ch/blog/experimenting-with-webmentions/ --debug
Sending webmention from https://adactio.com/journal/6469 to https://jaenis.ch/blog/experimenting-with-webmentions/... 
$VENV/venv/lib/python3.6/site-packages/urllib3/connectionpool.py:858: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
$VENV/lib/python3.6/site-packages/urllib3/connectionpool.py:858: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
Failed
{'code': 'RECEIVER_ERROR',
 'http_status': 201,
 'location': 'https://webmention.io/jaenis.ch/webmention/rQyV-tUujWlWQTnQc-qU',
 'request': 'POST https://webmention.io/jaenis.ch/webmention (with '
            'source=https://adactio.com/journal/6469, '
            'target=https://jaenis.ch/blog/experimenting-with-webmentions/)',
 'source': 'https://adactio.com/journal/6469',
 'status': 'queued',
 'summary': 'Webmention was queued for processing',
 'target': 'https://jaenis.ch/blog/experimenting-with-webmentions/'}
```

works now, that is, it shows up in https://webmention.io/dashboard - I consider it runnable now.

**Comment by Ryuno-Ki** [comment]
Hi @vrypan,

before I can make a call on that I'd need some more info on how to interact with PyPI etc.

My plan is at least to use WebMentions (and other IndieWeb goodies), but I haven't engaged with the Python community yet.

Do you have some resources to get me started?

**Comment by Ryuno-Ki** [comment]
Hi, @vrypan,

a colleague explained to me, what's necessary for PyPI. So it doesn't sound too complicated.
What do you need to transfer ownership? (GitHub shouldn't be too complicated).

Kind regards

**Comment by vrypan** [comment]
Hi @Ryuno-Ki,

There is an option in github to transfer ownership of a repository to an other user. You will receive an email to accept or reject the transfer.

However, you must not have a fork of the repo in your account. So you may have to delete your github repository (renaming will not work). I guess that once the transfer is complete, you can merge your local changes to the origin.

**Comment by Ryuno-Ki** [comment]
Hi @vrypan,
I've [read the docs](https://help.github.com/en/articles/transferring-a-repository) and deleted my fork (I have a local copy of it).
Especially the note, that I only have one day for reacting on the transfer request is important.
I'll be available for the next seven days. Please trigger the process within this timeframe if possible.

Let's do this! :-)

**Comment by Ryuno-Ki** [comment]
Link expired (in less than 24h)

**Comment by Ryuno-Ki** [comment]
Yay, that worked!
Thank you @vrypan!
I'll take good care on this and will close it once I published a new version on PyPI.

**Comment by Ryuno-Ki** [comment]
Closing in favour of #66
